data/api/dataloader_ready_config.md

```diff
@@ -1,21 +1,24 @@
 ready
-=============
+=====
 
 
-@short: a handler that is called just after the page has been completely parsed
+@short: Event handler called just after the component has been completely initialized.
 	
 
 @type:  function
 @example:
-
 webix.ui({
-	view:"list",
-	ready:function()
-    {
-		webix.message("Ready!");
-        //or any custom function
-	},
-	url:"data.json"
+  view: "tree",
+  data: function () {
+    webix.message("data");  // displayed first
+    return [
+      { id: "1", value: "Branch", webix_kids: true }
+    ]
+  }(),
+  ready: function() {
+    webix.message("ready");  // displayed second
+    this.data.url = "data/data_dyn.php";  // set the DataStore URL without triggering data loading
+  },
 });
 
```

Comment:
This is not the typical usage of ready and data properties while samples should reflect the typical ones. 
The code you added should be moved to the @descr: section of an article with a comment on data loading.